rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,12 +16,12 @@ toc::[]
 [id="rosa-q1-2024_{context}"]
 === Q1 2024
 
+* **ROSA CLI update.** The ROSA CLI (`rosa`) was updated to a new version. For information about what has changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.36[ROSA CLI release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-about_rosa-getting-started-cli[About the ROSA CLI].
+
 * **Log linking is enabled by default** - Beginning with {product-title} 4.15, log linking is enabled by default. Log linking gives you access to the container logs for your pods.
```

Comment:
Your actual addition looks good, I just noticed that this previously added bullet here is using a `-` instead of a `.` like the others. Maybe sync up with this PR?

```suggestion
* **Log linking is enabled by default.** Beginning with {product-title} 4.15, log linking is enabled by default. Log linking gives you access to the container logs for your pods.
```

Reply:
Thank you, this has been fixed!